### DISCUSSION TITLE
Classical Music  - Tags

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#444] [Ideas]
JC please bring these tags to myMPD. It would be very helpful for all the "classical" music friends out there :)

This is the PR to MPD with the new tags:

https://github.com/MusicPlayerDaemon/MPD/pull/858/commits/8f1e7385b71ee3f8e6140a22fbd74d6361f414d5

Thank you!

**Comment by jcorporation** [comment]
myMPD supports all tags provided by MPD out of the box. Only the mapping for ComposerSort must be manually added. I will add it to the v6.12.1 release.

**Reply by Emilot** [reply]
Great! Thanx JC

**Reply by Emilot** [reply]
Hi JC. I open "Tags to use" in Settings but i couldn't find the new tags! 

myMPD 6.12.1

Thx!

**Reply by jcorporation** [reply]
Are the tags enabled in MPD? Tags to use shows all tags that mpd announces.

**Reply by Emilot** [reply]
LOL..I forgot that small thing!! Ok. i will compile it and if i have a problem(i do not think so) i will come back.

Sorry JC...
